src/app.scss

```diff
@@ -11,11 +11,11 @@
 
   > .logo {
     grid-area: logo;
-    background-color: $color-black;
+    // background-color: $color-black;
```

Comment:
did you confirm that this commented-out code does not break anything?

would be better to just remove the lines than comment them out.

Reply:
I checked that this did not break anything the logo's background was set to black and I just remove that to use the containing divs background color but i could remove them too